### DISCUSSION TITLE
Login error

### DISCUSSION THREAD

**Discussion by MartinPrechtel** [orestbida/cookieconsent#345] [Q&A] [answered]
Hi,
I have installed the demo version for testing.
After confirming the cookie, I can no longer log in? Any idea.
Thanks in advance

**Comment by orestbida** [comment, marked as answer]
Mhm, there is likely an error that might be blocking other scripts. I'd need to see the code/page to know exactly where the issue is.

Edit:

The issue occurs only when the plugin's cookie is saved; the web server doesn't "like" the cookie value. Enabling the `use_rfc_cookie` option, should fix it.

**Comment by MartinPrechtel** [comment]
Hi,
this is the address to test.
With this link you can see the normal website
https://www.trekking-schwarzwald.de/?scc=0

As a demo the website without JQuery
https://www.trekking-schwarzwald.de/login?scc=1

Login:
demo@demo.de
demo

Best regards
Martin

**Reply by orestbida** [reply]
I don't see the plugin, so I guess you've given up.

**Comment by MartinPrechtel** [comment]
Hello,
I'm still working on it. You have to take this link. The parameter scc=1 activates the plugin.

https://www.trekking-schwarzwald.de/login?scc=1

**Reply by orestbida** [reply]
My bad! 

The issue occurs only when the plugin's cookie is saved; the web server doesn't "like" the cookie value. Enabling the `use_rfc_cookie` option, should fix it. 

I also noticed there is already another cookie with the same name as the plugin's own cookie. Is this intentional?

**Comment by MartinPrechtel** [comment]
Wow,
thank you so much. It works.
